### ISSUE TITLE
[Accessibility- Image 1.2] Inappropriate attributes for article image

### ISSUE THREAD

**Issue by mbensalem** [Meeds-io/meeds#3749]
Environment: Tribe

#### Steps to Reproduce
1. Create an artcile and add an illustration to it
2. Publish it
3. Access to the article

#### Current behavior
- The illustration is decalred in a div with role="img" an aria-label equals to

1. the article title, if the illustration doesn't have an alternative text
2. The alternative text if the illustration has an alternative text

#### Expected behavior

- If the image is decorative:  Delete role="img" and aria-label
- If the image is informative: It's better to use img + alt and avoid background-image (otherwise, there won't be a native equivalent if CSS is disabled).

**PS:** It's better to use img + alt for the 2 cases
